data/transactions/aggregates.go

```diff
@@ -37,6 +37,10 @@ type (
 // depends on the order in which the Txids appear, and that Txids do NOT cover
 // transaction signatures.
 func (payset Payset) Commit(flat bool) crypto.Digest {
+	if len(payset) == 0 {
```

Comment:
I think a comment explaining this needed

Reply:
Yeah -- and I might not actually be able to do this, because it looks like `makeGenesisBlock` actually calls `Commit` on a non-`nil` but empty payset...

Reply:
Added a comment and confirmed that nodes can still make progress from genesis

Reply:
(see comment regarding `makeGenesisBlock` below)